tests/xharness/Jenkins/Jenkins.cs

```diff
@@ -31,34 +61,14 @@ class Jenkins {
 		
 		public bool Populating { get; private set; } = true;
 
+		public TestSelection TestSelection { get; set; } = DefaultTestSelection;
+		internal const TestSelection DefaultTestSelection =  TestSelection.MacOS | TestSelection.iOS | TestSelection.tvOS |
+		                                                   TestSelection.watchOS | TestSelection.iOSMSBuild |
+		                                                   TestSelection.Simulator | TestSelection.NonMonotouch |
+		                                                   TestSelection.Monotouch;
+
 		public IHarness Harness { get; }
-		public bool IncludeAll;
-		public bool IncludeBcl;
-		public bool IncludeMac = true;
-		public bool IncludeiOS = true;
-		public bool IncludeiOS64 = true;
-		public bool IncludeiOS32 = true;
-		public bool IncludeiOSExtensions;
 		public bool ForceExtensionBuildOnly;
-		public bool IncludetvOS = true;
-		public bool IncludewatchOS = true;
-		public bool IncludeMmpTest;
-		public bool IncludeiOSMSBuild = true;
-		public bool IncludeMtouch;
-		public bool IncludeBtouch;
-		public bool IncludeMacBindingProject;
-		public bool IncludeSimulator = true;
-		public bool IncludeOldSimulatorTests;
-		public bool IncludeDevice;
-		public bool IncludeXtro;
-		public bool IncludeCecil;
-		public bool IncludeDocs;
```

Comment:
You could even add a setter here, that would make the `enable-docs-tests` changes simpler:

```csharp
public bool IncludeDocs {
    get { return (TestSelection & TestSelection.Docs) == TestSelection.Docs; }
    set {
        if (value) {
            TestSelection |= TestSelection.Docs;
        } else {
            TestSelection &= ~TestSelection.Docs;
        }
    }
}
```